frontend/src/lib/search.ts

```diff
@@ -16,6 +16,17 @@ export type CourseData = {
   classSection: string;
   component: string;
   description: string;
+  genEdAreas: {
+    hsi: boolean;
+    sts: boolean;
+    ac: boolean;
+    ec: boolean;
+  }[];
+  divisionalDistribution: {
+    ah: boolean;
+    ss: boolean;
+    sci: boolean;
+  }[];
```

Comment:
I think the type is wrong, it's just `string[]`

Reply:
Yup, sorry about that! A relic from an old way I was doing it that I forgot to fix.